thunder/core/functionalization.py

```diff
@@ -178,7 +178,7 @@ def replace_args_with_alias_map(
             swap_map_for_aliases[variableify(arg_to_replace)] = reshaped_arg
     for bsym in computation_trace.bound_symbols:
         for arg in filter(lambda p: isinstance(p, TensorProxy), bsym.flat_args):
-            if v_arg := variableify(arg) in arg_to_optional_bsyms:
+            if (v_arg := variableify(arg)) in arg_to_optional_bsyms:
```

Comment:
How about spelling it out instead, given that the only effect of `:=` is to save a single line of code
```suggestion
            v_arg = variableify(arg)
            if v_arg in arg_to_optional_bsyms:
```